app/addons/documents/queryoptions/queryoptions.react.jsx

```diff
@@ -343,6 +347,10 @@ A key value is the first parameter emitted in a map function. For example emit("
       Actions.toggleQueryBarVisibility(!this.props.contentVisible);
     },
 
+    closeTray: function () {
+      Actions.toggleQueryBarVisibility(false);
```

Comment:
Those are actually okay. The other occurrences of that method are specific to the standard query tray used in Fauxton, so that's fine: they can keep their own event handler. 

This ticket's just so I can re-use the QueryButtons component elsewhere.